sys/sys/ptrace.h

```diff
@@ -103,16 +103,16 @@
 
 struct ptrace_io_desc {
 	int	piod_op;	/* I/O operation */
-	void	*piod_offs;	/* child offset */
-	void	*piod_addr;	/* parent offset */
+	void	* __kerncap piod_offs;	/* child offset */
+	void	* __kerncap piod_addr;	/* parent offset */
 	size_t	piod_len;	/* request length */
 };
 
-#if defined(_KERNEL) && __has_feature(capabilities)
-struct ptrace_io_desc_c {
+#if defined(_KERNEL) && COMPAT_FREEBSD64
+struct ptrace_io_desc64 {
```

Comment:
Why do we need this for freebsd64 yet not have a corresponding freebsd32 version? This definition is also wrong, you should never have `void *` in freebsd64 structs.